docs/source/quickstart.rst

```diff
@@ -3,117 +3,162 @@
 ****************
 Quickstart Guide
 ****************
+The following sections describe, in short with examples, how to setup a few minimal cocotb testcases, how to run the simulation and how to view the generated waveform. For a more thorough and complete explanations about some of the intricacies of cocotb testbenches refer to the :ref:`writing_tbs`.
```

Comment:
```suggestion
The following sections describe, in short with examples, how to setup a few minimal cocotb testcases, how to run the simulation and how to view the generated waveform.
For a more thorough and complete explanations about some of the intricacies of cocotb testbenches refer to the :ref:`writing_tbs`.
```
We try to follow semantic linebreaks (https://rhodesmill.org/brandon/2012/one-sentence-per-line/) but are not strict about it.